### ISSUE TITLE
Mabye type for union types missing `namespacedImportName` prefix

### ISSUE THREAD

**Issue by GreenGremlin** [dotansimha/graphql-code-generator#5101]
**Describe the bug**
The `Maybe` types generated by the typescript-resolvers plugin are missing the import type prefix for union types.

**To Reproduce**

Generate resolver type definitions using the typescript-resolvers plugin for the following schema.

```graphql
type MyType {
  foo: String!
  unionChild: ChildUnion
}

type Child {
  bar: String!
  parent: MyType
}

type MyOtherType {
  bar: String!
}
```

With the following config

```yaml
  generated/resolvers.ts:
    preset: import-types
    presetConfig:
      typesPath: ./types
    plugins:
      - typescript-resolvers
```

The `Maybe` types for the union types should be `Types.Maybe`, but instead they're just `Maybe`.

**Comment by dotansimha** [comment]
Might also be related:
 - https://github.com/dotansimha/graphql-code-generator/issues/4901
- https://github.com/dotansimha/graphql-code-generator/discussions/4877

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/flow-resolvers@1.17.13` and `@graphql-codegen/typescript-resolvers@1.18.1`. Thanks @GreenGremlin
